service/models.py

```diff
@@ -60,8 +60,7 @@ class Product(db.Model):
     price = db.Column(db.Float(), nullable=False)
     available = db.Column(db.Boolean(), nullable=False, default=False)
     rating = db.Column(db.Float, nullable=True)
-    cumulative_ratings = db.Column(db.Integer, default=0)
-    no_of_users_rated = db.Column(db.Integer, default=0)
+    no_of_users_rated = db.Column(db.Integer, nullable=False, default=0)
```

Comment:
Haha you did delete the cumulative_ratings variable this time

Reply:
Yes! I have been bothered by this for a long time, lol

Reply:
Finally !!